src/DI/TranslationExtension.php

```diff
@@ -82,6 +82,9 @@ public function getConfigSchema(): Schema
 			'translatorFactory' => Expect::string()->default(null),
 			'returnOriginalMessage' => Expect::bool()->default(true),
 			'autowired' => Expect::type('bool|array')->default(true),
+			'hook' => Expect::structure([
```

Comment:
can you change to `latteFactory` (string|null) with default value `ILatteFactory::class`?

Reply:
changed to `latteFactory`